### PR TITLE
bazel: Fix build for Linuxbrew

### PR DESCRIPTION
**Pull request by iMichka**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/linuxbrew-core/blob/master/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/linuxbrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?
- [ ] Have you included the output of `brew gist-logs <formula>` of the build failure if your PR fixes a build failure. Please quote the exact error message.

-----

### CODE REVIEW

**Comment by aaronjheng** [comment]
Any updates ? Really looking forward.

**Comment by dawidd6** [comment]
Would really use some help here.

File: Formula/bazel.rb [hunk 1]
```diff
@@ -22,9 +22,17 @@ def install
     ENV["EMBED_LABEL"] = "#{version}-homebrew"
     # Force Bazel ./compile.sh to put its temporary files in the buildpath
     ENV["BAZEL_WRKDIR"] = buildpath/"work"
+
+    java_home = if OS.mac?
+      Formula["openjdk@11"].opt_libexec/"openjdk.jdk/Contents/Home"
+    else
+      Formula["openjdk@11"].opt_prefix
```
Comment (aaronjheng) [inline Formula/bazel.rb:29]: `Formula["openjdk@11"].libexec` should be good.

**Comment by aaronjheng** [comment]
@iMichka I'm not familiar with Homebrew Formula and Bazel. I can think of two ways as follows.

1. Use `Formula["openjdk@11"].libexec`.
2. Tweak openjdk@11 formula (`linux/jni_md.h` is missing in `Formula["openjdk@11"].opt_libexec/include`).

For now,  option 2 is better as for me. I hope this can help you.

**Comment by thoughton** [comment]
Please don't close, would be great if this could be fixed.

**Comment by iMichka** [comment]
Nobody is working on this.

**Pull request closed without merging**